crates/prism-recipe/harness/prismlib/deps.py

```
@@ -0,0 +1,162 @@
+        return None
+    kind, path = found
+    timeout_s = int(timeout_s or DEFAULT_INSTALL_TIMEOUT_S)
+    cmd = build_install_cmd(kind, path)
+    env = _cuda_build_env()
+    nvcc = shutil.which("nvcc", path=env.get("PATH"))
+    logfn(
+        f"[deps] installing miner {kind} manifest ({path}) timeout={timeout_s}s "
+        f"cuda_home={env.get('CUDA_HOME')} nvcc={nvcc or 'missing'}"
+    )
+    try:
+        proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)